PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchSavedItem+ItemsListItem.swift

```diff
@@ -35,11 +35,11 @@ extension SearchSavedItem: ItemsListItem {
     }
 
     var displayDomain: String? {
-        item.asItem?.domainMetadata?.name ?? item.asItem?.domain ?? host
+        item.asItem?.preview?.domain?.name ?? item.asItem?.preview?.domain?.name ?? host
```

Comment:
We should only need `item.asItem?.preview?.domain?.name`

Reply:
right thanks!